src/common/components/cards/CardCreationForm.js

```diff
@@ -60,10 +60,22 @@ export default class CardCreationForm extends Component {
       this.setState({
         bigramProbs: prepareBigramProbs(corpus)
       });
-      exampleStore.loadExamples(examples, 100);
+      exampleStore.onLoadExamples =
+      exampleStore.loadExamples(examples, 100, (mode) => {
+        this.setState(state => ({
```

Comment:
Kinda weird that you put `mode` in parentheses but not `state`. I tend to favor putting parentheses around all arrow function parameters.